docs/UserGuide.md

```diff
@@ -51,11 +58,11 @@ To view the user guide for the timetable mode, type in `help timetable`.
 To view the user guide for the notebook mode, type in `help notebook`.
 
 ### Switching between the two modes : `mode`
-Switches from any modes into the notebook mode (at the bookshelf level) or the timetable mode. 
+Switches from any modes into the notebook mode (at the bookshelf level) or the timetable mode.
```

Comment:
Is it better to phrase this as "Switches from notebook mode (at the bookshelf level) to the timetable mode or vice versa:. Since there are no other modes currently.

Reply:
Sure, I'll edit that before merging.